ar_exercise/exercise.rb

```diff
@@ -34,16 +34,23 @@ class Apartment < ActiveRecord::Base
 all_tenants = Tenant.all
 
 # get the first tenant in the DB
+first_tenant = Tenant.find(1)
 # get all tenants older than 65
+ova_65 = Tenant.where('age > 65')
```

Comment:
Did this work? When combining `.where` with a conditional, I usually use this syntax: http://stackoverflow.com/a/11317741